### DISCUSSION TITLE
[BE201] body-parser 套件已被標記為棄用(body-parser was marked as deprecated)

### DISCUSSION THREAD

**Discussion by KW0409** [Lidemy/forum#178] [後端相關問題] [answered]
今天在使用 `npm install`  安裝的 `body-parser` 的套件時
碰到這樣的狀況

<img src="https://user-images.githubusercontent.com/80152099/128891398-b161c2c9-2bf6-470c-b484-563b60fa03f4.png"/>

表示 body-parser 已經被淘汰，所以不能這樣使用
這邊分享一下找到的解決辦法

**Comment by KW0409** [comment, marked as answer]
如果是使用 Express v4.x 之後的版本，應該基本上都會碰到這個問題
文章在這邊--> [解決 body-parser 被標記為棄用(body-parser as deprecated)](https://hsiangfeng.github.io/nodejs/20210326/2926076225/) 請各位慢用 
如果有錯誤的地方，再請各位多多指教 感謝～

**Reply by ddylanlinn** [reply]
感謝
